multitenancy/tenant-manager-api/src/main/resources/application.properties

```diff
@@ -28,10 +28,10 @@
 
 %prod.quarkus.flyway.locations=db/migration/postgresql
 
-quarkus.hibernate-orm.database.default-schema=tenantmanager
+quarkus.hibernate-orm.database.default-schema=TENANTMANAGER
```

Comment:
I think I put this in lowercase because otherwise it did not work

Reply:
Interesting.  It definitely didn't work for me as lowercase.  It **only** worked as uppercase. :(  Can you test locally and see what you get?  When running in Windows it 100% mattered and it needed to be uppercase for sure.

Reply:
actually it's failing for the same reason in GH actions https://gist.github.com/famartinrh/aaa1eef6aacca56bac4807ba8e83e2df

Reply:
it's like hibernate makes the schema lowercase for some queries but not for some others

Reply:
I could revert this change, but it will definitely start to fail for me locally.  Any idea on how to address this?